core/templates/dev/head/components/subtitled_html_editor_directive.html

```diff
@@ -0,0 +1,7 @@
+<div ng-if="!subtitledMemento.htmlContent.isEmpty() && subtitledMemento.htmlContent">
+  <audio-translations-editor component-name="<[COMPONENT_NAME_HTMLCONTENT]>"
+                             subtitled-html="subtitled.htmlContent"
+                             on-start-edit="onAudioTranslationsStartEditAction"
+                             on-change="onAudioTranslationsEdited">
+  </audio-translations-editor>
+</div>
```

Comment:
There should be a newline at the end of the file. I wonder why lint tests didn't catch that.

Reply:
I think it is contain an extra line

Reply:
oh, ok. I didn't notice that before.